EIPS/eip-7525.md

```diff
@@ -0,0 +1,161 @@
+---
+eip: 7525
+title: Expire state of the inactive accounts
+description: Expire the status of inactive accounts after a certain period.
+author: Dan Park (@coldpak) <dan@over.network>, Ben Kim <ben@over.network>, Jade Shin <jade@over.network>, et al.
+discussions-to: https://ethereum-magicians.org/t/add-eip-expire-state-of-the-inactive-accounts/15904
+status: Draft
+type: Standards Track
+category: Core
+created: 2023-09-25
+---
+
+## Abstract
+
+In the Execution Layer, accounts that have been inactive for a certain period are expired. The expired state can be recovered through a special transaction for restoration. This significantly lowers the entry barrier for node operation by maintaining only the minimal required state for validation.
+
+## Motivation
+
+Numerous attempts to create a stateless roadmap exist, but we still lack practical solutions, and the state size keeps increasing quickly. Recently, researchers at a16z pointed out the significant challenges in maintaining a stateless chain, due to the heavy load of witness updates. Given these hurdles, we believe that developing mechanisms for state expiry is crucial for the community.
+
+According to our analysis, we have observed that 95% of the accounts in Execution Layer have been inactive for at least one month. The inactive accounts refer to those that have not been in engaged or affected by any transactions.
```

Comment:
any reference to analysis would be nice

Reply:
I tried to make proper references to those, but for a16z's research, the DOI has not been registered yet, and external links have been abandoned by the 'ethw' linter. As for our analysis section, the markdown linter has been causing issues, and I couldn't find a proper way to resolve it without using magic keywords like:

<!-- markdownlint-capture -->
<!-- markdownlint-disable code-block-style -->
Do you happen to know how to add a DOI without triggering a warning?

Reply:
you could just screenshot diagrams and link, would be very nice context. what DOI are you registering share here and let me see

Reply:
This is [our paper](https://dl.acm.org/doi/10.1145/3447786.3456231) about state expiry 

And this is an a16z's recent [article](https://a16zcrypto.com/posts/article/on-the-impossibility-of-stateless-blockchains/) and [research paper](https://eprint.iacr.org/2022/1478
) about "On the impossibility of stateless blockchains"